cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -383,6 +383,36 @@
           "widget-attributes" : {}
         }
       ]
+    },
+    {
+      "label": "Experimental Features (BETA)",
+      "properties": [
+        {
+          "name": "autoScaling",
+          "widget-type": "toggle",
+          "label": "Enable Autoscaling",
+          "widget-attributes": {
+            "on": {
+              "value": "true",
+              "label": "True"
+            },
+            "off": {
+              "value": "false",
+              "label": "False"
+            },
+            "default": "false"
+          }
+        },
+        {
```

Comment:
This should be conditional only when it's 'Enable' is selected.

Reply:
The UI Filter framework does not work in the Provisioner UI.  I tried this during my Existing Dataproc task and wanted to implement it but it failed.